tests/helics/application_api/FederateInfoTests.cpp

```diff
@@ -163,11 +165,11 @@ TEST(federateInfo, property_index)
 TEST(federateInfo, option_index)
 {
     EXPECT_EQ(helics::getOptionIndex("unexpected"), -1);
-    EXPECT_EQ(helics::getOptionIndex("single_connection"),
+    EXPECT_EQ(helics::getOptionIndex("single_connection_only"),
               helics_handle_option_single_connection_only);
-    EXPECT_EQ(helics::getOptionIndex("StrictTypeChecking"),
+    EXPECT_EQ(helics::getOptionIndex("strictInputTypeChecking"),
               helics_handle_option_strict_type_checking);
-    EXPECT_EQ(helics::getOptionIndex("un_interruptible"), helics_handle_option_ignore_interrupts);
+    EXPECT_EQ(helics::getOptionIndex("uninterruptible"), helics_handle_option_ignore_interrupts);
```

Comment:
```suggestion
    EXPECT_EQ(helics::getOptionIndex("un_interruptible"), helics_handle_option_ignore_interrupts);
```

This was testing the ignore underscore